resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -218,3 +218,220 @@
       re-save-file-list fastqc_archive2 "${RW_FASTQC[@]}"
       re-save-list fastqc_url "${FW_FASTQC_URL[@]}"
       re-save-list fastqc_url2 "${RW_FASTQC_URL[@]}"
+
+- slug: basespace-fastq-single
+  name: BaseSpace Reads (single-end)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq
+  data_name: 'BaseSpace FASTQ single-end {{ reads.files.0.file }}'
```

Comment:
Put the filename also in parentheses (round brackets), like in [Cutadapt](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/reads_processing/cutadapt.yml#L15).